### ISSUE TITLE
Invitation doesn't contain reminder

### ISSUE THREAD

**Issue by schiessle** [nextcloud/calendar#5366]
### Steps to reproduce

1. Setup a invitation with the Nextcloud calendar
2. make sure to set a reminder

![grafik](https://github.com/nextcloud/calendar/assets/1589737/b779c865-f014-49da-b2a4-e1904e101cb7)

3. send out the invitation

### Expected behavior

The recipient gets a invitation with the reminder set

### Actual behaviour

The recipient gets a invitation without a reminder:

````
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//Sabre//Sabre VObject 4.4.2//EN
CALSCALE:GREGORIAN
METHOD:REQUEST
BEGIN:VTIMEZONE
TZID:Europe/Oslo
BEGIN:DAYLIGHT
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
DTSTART:19700329T020000
RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
END:DAYLIGHT
BEGIN:STANDARD
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
DTSTART:19701025T030000
RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
END:STANDARD
END:VTIMEZONE
BEGIN:VEVENT
CREATED:20230717T212315Z
LAST-MODIFIED:20230717T212836Z
SEQUENCE:2
UID:9f250548-4f25-4b92-b004-7ac7e3f188a2
DTSTART;TZID=Europe/Oslo:20230721T063000
DTEND;TZID=Europe/Oslo:20230721T070000
STATUS:CONFIRMED
SUMMARY:Test4
ATTENDEE;CN=bjoern@nextcloud.com;CUTYPE=INDIVIDUAL;PARTSTAT=NEEDS-ACTION;RO
 LE=REQ-PARTICIPANT;RSVP=TRUE:mailto:bjoern@nextcloud.com
ORGANIZER;CN=Björn Schießle:mailto:bjoern.schiessle@nextcloud.com
DTSTAMP:20230717T212836Z
END:VEVENT
END:VCALENDAR
````

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

**Comment by schiessle** [comment]
@tcitworld you are right. I remember that for example Evolution asks the user if they want to send a invitation with or without the reminder if the invitation is send client side.

If we remove it at the moment on purpose we might consider a calendar setting? A setting per invite when I create a appointment doesn't work because invitations can also be triggered from outside (desktop/mobile calendars)

I would love to have a setting or even default to send invites with the reminder. People ask me regularly why my invitations doesn't contain a reminder.

**Comment by tcitworld** [comment]
Can't check it right now, but I guess there's some information on the topic in the iMip/iTip/CalDAV RFCs.
